Src/GoogleApis.Auth/OAuth2/UserCredential.cs

```diff
@@ -91,17 +91,9 @@ public UserCredential(IAuthorizationCodeFlow flow, string userId, TokenResponse
         /// minute away from expiration. If token server is unavailable, it will try to use the access token even if 
         /// has expired. If successful, it will call <see cref="IAccessMethod.Intercept"/>.
         /// </summary>
-        public async Task InterceptAsync(HttpRequestMessage request, CancellationToken taskCancellationToken)
+        public async Task InterceptAsync(HttpRequestMessage request, CancellationToken cancellationToken)

```

Comment:
i noticed there's an inconsistency between UserCredential and other credential types. That's fine, I'll remove the change.